### PR TITLE
Changed MiniSat installation directory from /usr to /usr/local and mo…

### PR DESCRIPTION
**Pull request by ccadar**
…ved note about STP dependencies just after the command for installing the dependencies.

### CODE REVIEW

File: build-stp.md [hunk 1]
```diff
@@ -8,20 +8,23 @@ slug: build-stp
...
-_Please let us know if you have successfully and extensively used KLEE with a more recent version of STP._  
+_Please let us know if you have successfully and extensively used KLEE with a more recent version of STP._
 
 STP has a few external dependencies they are listed below as an install command for Ubuntu 14.04LTS.
 
 ```bash
 sudo apt-get install cmake bison flex libboost-all-dev python perl zlib1g-dev
 ```
 
+**NOTE:** If you are using an older Linux release (e.g. Ubuntu ≤12.04), then you will have to manually install cmake 2.8.8 or newer (you can follow the instructions [here](http://cameo54321.blogspot.com/2014/02/installing-cmake-288-or-higher-on.html)).
```
Comment (delcypher) [inline build-stp.md]: Please just link to the official documentation. https://cmake.org/install/ .

Comment (delcypher) [inline build-stp.md]: Also you should recommend a newer version CMake version. STP requires CMake >= 3.0.2, KLEE requires >= 2.8.12

File: Gemfile.lock [hunk 2]
```diff
@@ -142,4 +142,4 @@ DEPENDENCIES
   github-pages (>= 104)
 
 BUNDLED WITH
-   1.13.6
+   1.15.1
```
Comment (delcypher) [inline Gemfile.lock]: Please remove this. There shouldn't be any changes to this file.

Comment (ccadar) [inline Gemfile.lock]: Good catch.

File: build-stp.md (hunk 1 above)
Comment (ccadar) [inline build-stp.md]: I didn't change anything there, just moved to a different location.  Please create a different PR for that.

**Comment by ccadar** [comment]
Removed the Gemfile.lock file, should be ready to merge

**Pull request merged**